### PR TITLE
Feature/4885 white grey bg

### LINKED ISSUE #4885: Alternate white/grey background for sections on the new homepage
This is blocked until the front end work of the new homepage is done.

---

Alternate white/grey background for sections on the new homepage. See mockup: https://redpen.io/sb271478cae0b6a0f8

### PR DESCRIPTION
**Pull request by KalobTaulien**
Closes #4885 

I moved more of the homepage sections into their own fragments to make this ticket easier to manage.

### CODE REVIEW

File: network-api/networkapi/wagtailpages/templates/wagtailpages/homepage.html [hunk 1]
```diff
@@ -31,172 +31,33 @@ <h1 class="h1-heading">{{ page.hero_headline }}</h1>
...
-        {% endfor %}
-      </div>
-    </div>
-  </div>
 <div class="container">
+  {% include "./fragments/spotlight_posts.html" %}
+  {# @TODO Confirm these are no longer required in the template. #}
+  {% comment %} {% include "./fragments/featured_blogs.html" %} {% endcomment %}
+  {% comment %} {% include "./fragments/featured_stories.html" %} {% endcomment %}
+  {# END @TODO #}
```
Comment (KalobTaulien) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/homepage.html:47]: I want to highlight this: Do we need these still? The design _looks_ like we don't but I want to confirm before properly deleting it. @mmmavis

File: source/sass/wagtail.scss [hunk 2]
```diff
@@ -100,3 +100,7 @@
 }
 
 @import "./wagtail/blocks";
+
+.bg-gray {
+  background: $gray-05;
```
Comment (KalobTaulien) [inline source/sass/wagtail.scss:105]: @mmmavis Is there a better place to put this?

File: network-api/networkapi/wagtailpages/templates/wagtailpages/homepage.html (hunk 1 above)
Comment (mmmavis) [reply network-api/networkapi/wagtailpages/templates/wagtailpages/homepage.html:47]: Yeah looks like they are not needed anymore. But let's comment the template code out for now so they can act like a reminder for us to remove the rest of the related backend & CSS code.

**Comment by mmmavis** [comment]
@KalobTaulien thanks for splitting code into fragments. However, since tons of code got shifted, I'm having issues spotting the actual code change for the white/grey background. Can you point me to the places I should look at?

**Comment by KalobTaulien** [comment]
> @KalobTaulien thanks for splitting code into fragments. However, since tons of code got shifted, I'm having issues spotting the actual code change for the white/grey background. Can you point me to the places I should look at?

@mmmavis Definitely, sorry about that! It's in the homepage. So I split out the fragments but kept the containers and wrappers in homepage.html. [This would be an example](https://github.com/mozilla/foundation.mozilla.org/pull/5012/files#diff-7c8b6f2fc5247d9e519764de75d968efR37) and [this too](https://github.com/mozilla/foundation.mozilla.org/pull/5012/files#diff-7c8b6f2fc5247d9e519764de75d968efR51)

File: source/sass/wagtail.scss (hunk 2 above)
Comment (mmmavis) [reply source/sass/wagtail.scss:105]: Yup! I'm thinking `global.scss`

**Pull request merged**